app/controllers/sessions_controller.rb

```diff
@@ -94,4 +92,12 @@ def ensure_not_blocked
     flash.now.alert = t(".account_blocked")
     render template: "sessions/new", status: :unauthorized
   end
+
+  def mfa_recommended_not_yet_enabled?
```

Comment:
What do you think about this naming @jenshenny? I like the symmetry of `mfa_recommended` prefix and that they both speak to the level of how mfa is `enabled`. 

The alternates we had discussed were:
`mfa_disabled_on_recommended_account` & `weak_mfa_on_recommended_account`